Ch03/operation.cpp

```diff
@@ -0,0 +1,29 @@
+#include "../std_lib_facilities.h"
+
+int main() {
+	string operation;
+	double opd1;
+	double opd2;
+	bool tof = true;
+	cout << "This is a calculator. Please type in your operation, then the two operands, with spaces in between. ";
+	while (tof == true) {
+		cin >> operation;
+		cin >> opd1;
+		cin >> opd2;
+		if (operation == "+") {
+			cout << opd1+opd2 << "\n";
+			tof = false;
```

Comment:
tof is assigned to false in too many places causing code duplication. Can we initialize tof to false in line 7, then assign it to true in line 26? You will need to use do...while(...) instead of while(...), though.